docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,313 @@
+recommended to be used in the context of the Open Science movement, but it is not mandatory. However, one needs to
+collect this information to build the authors lists, in a safe and fair way.
+
+In this purpose, one could use the public user profile of the GitHub platform. They are public data, but one should
+aware that they are not complete and/or up-to-date. As mention earlier, it is recommended to fill correctly this GitHub
+profile to help the dev team to pre-filled the metadata files.
+
+One should note that the GitHub profile does not contain a field associated with the ORCID
+identifier. However, one could also retrieve the ORCID identifier of a contributor with the ORCID API
+(e.g. `How-To-1 <https://stackoverflow.com/questions/71052912/extracting-credentials-from-orcid-seach-with-orcid-id-using-python>`_
+or `How-To-2 <https://info.orcid.org/documentation/api-tutorials/api-tutorial-searching-the-orcid-registry/#easy-faq-2532>`_).
+
+In any case, a specific Python script located in the main Gammapy repository had to be written such
+that the authors' order follows these rules. In this case, integration within a GitHub action could be possible.
+
+In this context, one could technically use an other cff file as maintenance file to store the personal data of users
+that have signed the DCO.
+
+A second script should be settled in order to create the list of contributors in HTML format that should be
+inserted in the `team` section for our general web page. This script would read one of the metadata files (e.g.
+``citation.cff``) to create such list.
+
+It is important to note that the list of personal information might change from one editor to an other. For the code
+deposits in Zenodo, SWH, etc, the data follow standards of Open Science and scripts can be settled. For publications,
+the editors have frequently their own scheme and adaptations will be made on the case by case.
+
+As mentioned here, the authors list will be reviewed for any publication, for safety and to allow OptIn/OptOut requests.
+
+5.3 Handling of conference material
+-----------------------------------
+
+Conference contributions (Proceedings, Posters, ...) could be developed in dedicated repositories in the Gammapy Github
+organization, as well as the author list.
+
+6. Suggestions
+==============
+
+One could ask CTAO software coordinators (ie ACADA, DPPS, SUSS) if such rules can be used by them also, even if the
+The Gammapy project is independent. In the same spirit, advice could be asked to people of the ESCAPE project via our
+corresponding person. And finally, one could mention the existence of these rules and ask for advice to the Astropy
+project.
+
+These requests to the Astropy community and CTAO for recommendations and preferences either remained
+un-responded or did not lead to objections at the current time.
+
+7. Decision
+===========
+After the addition of comments and proposals, the PIG is accepted by the dev team and the CC.
```

Comment:
Maybe add a sentence about the choice of practical implementation to be decided later?